src/styles/main.scss

```diff
@@ -77,6 +77,10 @@ i {
   @include btn($btn-active, $btn-active-border);
 }
 
+.btn-inactive {
+  @include btn($btn-inactive, $btn-inactive-border, default, .4, false); 
```

Comment:
What about `.5` for opacity? It's hard to see the text on the button. At least for my computer 😬